### PR TITLE
Fix formatting in generated `request.js`

### LINKED ISSUE #29: `request.js` spacing and formatting nitpicks
image
image
Prettier runs in our repo, and always updates the files because the spacing is a little off from `snippets.js`; also missing semicolons.

### PR DESCRIPTION
**Pull request by annieyro**
Closes #29. Nit-picky, but formatting is off in the current generated `request.js`, so adjusted all spacing, added semicolons, added a newline to the file, etc.

### CODE REVIEW

File: lib/generators.js [hunk 1]
```diff
@@ -30,9 +30,9 @@ function generateRequestFile(schema, settings) {
 
   // For each CRUD function, call the corresponding snippet function on each table in schema
   // and append to fileContents.
-  ['create', 'read', 'update', 'delete'].forEach(fn => {
+  ['create', 'read', 'update', 'delete'].forEach((fn) => {
```
Comment (aivantg) [inline lib/generators.js:33]: i’m seeing some differences in our eslint configs here LOL

File: lib/snippets.js [hunk 2]
```diff
@@ -117,13 +120,13 @@ export const get${pluralize(
     return result;
   },
   updateRecord: (tableName) => `
-export const update${tableName} = async (id, recordUpdates) => { 
+export const update${tableName} = async (id, recordUpdates) => {
```
Comment (aivantg) [inline lib/snippets.js:123]: hm maybe we should like....just run prettier or something in all outputted files..... but as long as it works i’m down!

Comment (aivantg) [reply lib/snippets.js:123]: LOL this comment was meant for the earlier edits about formatting

**Review by aivantg — APPROVED** [review]

File: lib/generators.js (hunk 1 above)
Comment (annieyro) [reply lib/generators.js:33]: wait I feel like this got changed on your side in #34 😂

File: lib/snippets.js (hunk 2 above)
Comment (annieyro) [reply lib/snippets.js:123]: (when testing this I realized prettier was running on all my files anyway LOL so I had to turn it off but I guess consistency in the snippets or whatever?!)

**Pull request merged**